src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -172,13 +175,38 @@ private void SortByColumn(GridViewColumnHeader sortColumn)
             {
                 if (column == sortColumn)
                 {
+                    UpdateHeaderAutomationProperties(column);
                     continue;
                 }
 
                 SortableColumnHeaderAttachedProperties.RemoveSortDirectionProperty(obj: column);
+                UpdateHeaderAutomationProperties(column);
             }
         }
 
+        private void UpdateHeaderAutomationProperties(GridViewColumnHeader columnHeader)
+        {
+            var sortDir = SortableColumnHeaderAttachedProperties.GetSortDirectionProperty(columnHeader);
```

Comment:
why not just pass in the sortDir since we already have it?

Reply:
The difference is the sortDir in the calling function is the desired sort direction for the column that is being modified while the sortDir in the UpdateHeaderAutomationProperties is for each column.